src/VisualStudio/Core/Def/Implementation/MoveToNamespace/MoveToNamespaceDialog.xaml

```diff
@@ -0,0 +1,90 @@
+             xmlns:imaging="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.Imaging"
+             xmlns:system="clr-namespace:System;assembly=mscorlib"
+             mc:Ignorable="d"
+             MinWidth="400" MinHeight="166"
+             Width="400" Height="166"
+             Title="{Binding ElementName=dialog, Path=MoveToNamespaceDialogTitle}"
+             ResizeMode="CanResizeWithGrip"
+             ShowInTaskbar="False"
+             HasDialogFrame="True"
+             WindowStartupLocation="CenterOwner">
+
+    <Window.Resources>
+        <Thickness x:Key="verticalLabelMargin">0, 0, 0, 5</Thickness>
+        <Thickness x:Key="messageMargin">0, 9, 0, 0</Thickness>
+        <Thickness x:Key="submitButtonSpacingMargin">6, 0, 0, 0</Thickness>
+        <system:Double x:Key="buttonWidth">75</system:Double>
+        <system:Double x:Key="buttonHeight">23</system:Double>
+        <BooleanToVisibilityConverter x:Key="BooleanToVisibilityConverter" />
+    </Window.Resources>
+
+    <Grid Margin="8, 0, 8, 12" x:Name="Root">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="25" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <StackPanel Orientation="Vertical" Grid.Row="0">
+            <Label x:Uid="NamespaceLabel" 
+                       x:Name="NamespaceLabel"
+                       Margin="{StaticResource verticalLabelMargin}"
+                       Content="{Binding ElementName=dialog, Path=NamespaceLabelText}"
+                       Target="{Binding ElementName=NamespaceTextBox}" />
+            <ComboBox x:Uid="NamespaceBox"
+                          Name="NamespaceBox"
+                          IsEditable="True"
+                          IsReadOnly="False"
+                          AutomationProperties.LabeledBy="{Binding ElementName=NamespaceLabel}"
+                          Text="{Binding NamespaceName, Mode=TwoWay, ValidatesOnDataErrors=True}"
+                          ItemsSource="{Binding AvailableNamespaces, Mode=OneTime}"
+                          BorderThickness="0" 
+                          MaxDropDownHeight="200" />
+        </StackPanel>
+
+        <StackPanel Grid.Row="1">
+            <StackPanel Orientation="Horizontal"  
+                        Visibility="{Binding ShowMessage, Converter={StaticResource BooleanToVisibilityConverter}}"
+                        Margin="{StaticResource messageMargin}">
+                <imaging:CrispImage Name="MessageIcon"
+                                    Moniker="{Binding Icon}" 
+                                    Height="16"
+                                    Width="16"
+                                    VerticalAlignment="Top" />
+
+                <vs:LiveTextBlock x:Name="MessageText"
+                           Text="{Binding Message}"
+                           AutomationProperties.LiveSetting="Assertive"
```

Comment:
I think it was suggested to remove "Assertive" in some mail thread... do you know what's the right thing to do there?

Reply:
Remove assertive, I just missed it